etc/db-config.yaml

```diff
@@ -392,6 +392,11 @@
             description: Time in seconds after an external contest source reader last checked in before showing its status as `critical`.
             regex: /^\d+$/
             error_message: A non-negative number is required.
+        -   name: check_new_version
+            type: bool
+            default_value: false
+            public: false
+            description: Automatically check for new DOMjudge release?
```

Comment:
```suggestion
            description: Automatically check and notify for new DOMjudge releases?
```

Reply:
Did you mark this as resolved because you don't like it (which is fine) or by accident?